public/app/features/explore/TraceView/components/TracePageHeader/TracePageHeader.tsx

```diff
@@ -176,20 +196,78 @@ export default function TracePageHeader(props: TracePageHeaderEmbedProps) {
     return { ...rest, value: renderer(trace, timeZone, styles) };
   });
 
+  const spanWithHeaderTags = getSpanWithHeaderTags(trace.spans);
+
   const title = (
     <h1 className={styles.TracePageHeaderTitle}>
       <TraceName traceName={getTraceName(trace.spans)} />{' '}
       <small className={cx(styles.TracePageHeaderTraceId, uTxMuted)}>{trace.traceID}</small>
     </h1>
   );
 
+  const newTitle = (
+    <h1 className={cx(styles.TracePageHeaderTitle)}>
+      <TraceName traceName={getTraceName(trace.spans)} />
+      <small>
+        <span className={styles.divider}>|</span>
+        {formatDuration(trace.duration)}
+      </small>
+    </h1>
+  );
+
+  const method = spanWithHeaderTags?.tags.filter((tag) => {
+    return tag.key === 'http.method';
+  });
+
+  const status = spanWithHeaderTags?.tags.filter((tag) => {
+    return tag.key === 'http.status_code';
+  });
+
+  let statusColor: BadgeColor = 'orange';
+  if (status && status.length > 0 && Number.isInteger(status[0].value)) {
+    if (status[0].value.toString().charAt(0) === '2') {
+      statusColor = 'green';
+    } else if (status[0].value.toString().charAt(0) === '4') {
+      statusColor = 'red';
+    }
+  }
+
+  const url = spanWithHeaderTags?.tags.filter((tag) => {
+    return tag.key === 'http.url' || tag.key === 'http.target';
+  });
+
   return (
     <header className={styles.TracePageHeader}>
-      <div className={styles.TracePageHeaderTitleRow}>
+      <div
+        className={cx(styles.TracePageHeaderTitleRow, !config.featureToggles.newTraceView && styles.titleBorderBottom)}
+      >
         {links && links.length > 0 && <ExternalLinks links={links} className={styles.TracePageHeaderBack} />}
-        {title}
+        {config.featureToggles.newTraceView ? newTitle : title}
       </div>
-      {summaryItems && <LabeledList className={styles.TracePageHeaderOverviewItems} items={summaryItems} />}
+
+      {config.featureToggles.newTraceView ? (
+        <div className={styles.subtitle}>
+          {timestamp(trace, timeZone, styles)}
+          <span className={styles.divider}>|</span>
+          {method && method.length > 0 && (
+            <span className={styles.tag}>
+              <Badge text={method[0].value} color="blue" />
```

Comment:
It would be nice to give all the items `title` attribute so that on hover you can still get some info about what the value represents now that we removed visible titles.

Reply:
Sorry not sure why we'd add a title to method/status when the text is only a few chars? Url below has a tooltip :)

Reply:
So for example in the data right now I have `app:80`. I wasn't even sure it was a URL at that point. I think it's easy and accessible way to just say that 200 badge is a `status code` for example. 

Also having tooltip for the URL being the URL value itself I assume is a mistake?

Reply:
Added a tooltip for `status` and `method`.

We had a tooltip for `url`, but that was for showing the full url if it was too long to be in the header itself (as don't want the header container to grow indefinitely so instead use a text with an ellipsis in the header and showed the whole url in the tooltip). 

However, for now, I'd like to remove the url from the tooltip. I do plan to revisit the url length as more of the reworked header/filtering work is done.